rust/extractor/src/rust_analyzer.rs

```diff
@@ -62,7 +62,7 @@ impl<'a> RustAnalyzer<'a> {
         match load_workspace_at(manifest.as_ref(), config, &load_config, &progress) {
             Ok((db, vfs, _macro_server)) => Some((db, vfs)),
             Err(err) => {
-                log::error!("failed to load workspace for {}: {}", manifest, err);
+                tracing::error!("failed to load workspace for {}: {}", manifest, err);
```

Comment:
here we can directly drop `tracing::` as far as I can see (we already pull in `error` above)